src/app/hooks/useImageColour/index.js

```diff
@@ -0,0 +1,51 @@
+import { useState, useEffect } from 'react';
+import Vibrant from 'node-vibrant';
```

Comment:
are we ok with this adding 60k on to the bundle size?

this is the current bundle size without this package:
<img width="555" alt="Screenshot 2021-11-17 at 13 05 49" src="https://user-images.githubusercontent.com/4798332/142206286-f74bc3de-0c75-4673-a30b-26af940fb001.png">

and this is with node-vibrant:
<img width="566" alt="Screenshot 2021-11-17 at 13 06 00" src="https://user-images.githubusercontent.com/4798332/142206313-913b2e35-30e7-40b2-9b5e-5a3af8d645b3.png">

60k seems a lot. I think that would make it the biggest library we use.

Reply:
yeah I definitely don't think this is worth it in general

We'd only load it for people in the experiment group though, which is probably tolerable - but I'll see how colorthief compares - https://www.npmjs.com/package/colorthief

Reply:
Yeh colorthief was much smaller and has a cooler name. Hopefully it's not troublesome to potentially swap out.